test/functional/developer_portal/admin/account/personal_details_controller_test.rb

```diff
@@ -63,4 +63,19 @@ def setup
     assert_response :success
     assert_equal flash[:error], 'Current password is incorrect'
   end
+
+  test 'changing password is audited' do
+    login_as @buyer.admins.first
+    assert_difference(Audited.audit_class.method(:count)) do
+      User.with_synchronous_auditing do
+        put :update, params: { user: {current_password: 'supersecret', password: 'new_password', password_confirmation: 'new_password'} }
```

Comment:
also asserting that the value of old and new password in audit is `[REDACTED]` could be useful

Reply:
Mmmm, actually...

```ruby
[27] pry(main)> user.audits.last
=> #<Audited::Audit:0x000000004247d5a0
 id: 1454,
 auditable_id: 2,
 auditable_type: "User",
 user_id: 2,
 user_type: "User",
 username: nil,
 action: "update",
 version: 2,
 created_at: Thu, 19 Sep 2024 12:02:25.000000000 UTC +00:00,
 tenant_id: 2,
 provider_id: 2,
 kind: "User",
 audited_changes: {"password_digest"=>"[FILTERED]"},
 comment: nil,
 associated_id: nil,
 associated_type: nil,
 remote_address: "127.0.0.1",
 request_uuid: "d97ee5f5-ccf1-4dc8-a39a-e4e2aec767e3">

[28] pry(main)> user.audits.last.audited_changes
=> {"password_digest"=>
  ["$2a$12$TshOAone/WSVbXsCmOB1JuLGBQI698CButPpA2F7hw1jEb2db4YWi",
   "$2a$12$.0G4VigmmeE1cjozQbXPkuyYF5iBjcfIwtjqcqJCPomceKUr7REkS"]}
```

Is that expected?

Reply:
Shit. It was actually not redacting the value. I'll fix it and add the test.

Reply:
Fixed: https://github.com/3scale/porta/pull/3885/commits/efa0fdf398c33d5eca8c77a11b2cf5eff7c31b3f

And updated the tests: https://github.com/3scale/porta/pull/3885/commits/5f93b2c880c0eef43e8e627c4fae0e83dab432a3